server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -364,6 +375,8 @@ public void fetchData() {
         loadOverTime.add(new Pair<>(currentTime, totalLoad));
 
         minorCompactionsOverTime.add(new Pair<>(currentTime, minorCompactions));
+        majorCompactionsOverTime
+            .add(new Pair<>(currentTime, getRunnningCompactions().running.size()));
```

Comment:
Not suggesting any changes here, just commenting on something I noticed. The getRunningCompactions() call will only update up to once per minute while this fetchData function could run up to every 5 seconds.  I don't think we want to fetch the running compactions any more frequently though.

Reply:
I noticed that too, and I agree that we don't want to make it more frequent. I left this way thinking that things might be a little out of sync, but I'm not sure how important that is.